CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ### 2.2.1 (Next)
 
+* [#92](https://github.com/acm19/aws-request-signing-apache-interceptor/pull/92): Make samples compatible with OpenSearch 2.x - [@dblock](https://github.com/dblock).
```

Comment:
Can you bump the testing cluster version as part of this PR? https://github.com/acm19/aws-request-signing-apache-interceptor/blob/master/infra/opensearch_cluster/opensearch_cluster.py#L13

Reply:
No can do :(

https://github.com/aws/aws-cdk/issues/22942